templates/user/edit.html

```diff
@@ -0,0 +1,73 @@
+{% from 'blog/macros.html' import article_list with context %}
+{% extends 'layout.html' %}
+
+{% block title %}My profile{% endblock %}
+
+{% block extra_css %}
+<link rel=stylesheet href={{ url_for('static', filename='css/edit.css') }}>
+<link rel=stylesheet href={{ url_for('static', filename='css/profile.css') }}>
+<link rel=stylesheet href={{ url_for('static', filename='css/_article_list.css') }}>
+{% endblock %}
+
+{% block main -%}
+  <header>
+    <form method=post action={{ url_for('profile_save') }} enctype=multipart/form-data>
+      <div class="profile-intro wrapper">
+        <figure>
+          <img class=avatar-img
+            name=avatar
+            src={{ user.avatar }}
+            width=100
+            height=100
+            onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'">
+          <caption>
+            <input type=file name=avatar>
+            {% if user.avatar %}
+            <label><input type=checkbox name=delete value=1>Remove this picture</label>
+            {% endif %}
+          </caption>
+        </figure>
+
+        <section>
+          <input name=username value="{{ user.username or '' }}" placeholder="Insert a username">
+          <textarea name=about placeholder="Tell a little about yourself">{{ user.about|htmlnewline }}</textarea>
```

Comment:
We need a defaut text in order to type in`{{ user.about|htmlnewline || "Tell a little about yourself" }}`
This is as a placeholder, indeed, but also to force a height.
Note that you can keep the placeholder anyways :)